cmd/nerdctl/volume_ls.go

```diff
@@ -73,6 +76,23 @@ func volumeLsAction(cmd *cobra.Command, args []string) error {
 		logrus.Warn("cannot use --size and --quiet together, ignoring --size")
 		volumeSize = false
 	}
+	filters, err := cmd.Flags().GetStringSlice("filter")
+	if err != nil {
+		return err
+	}
+	labelFilterFuncs, nameFilterFuncs, sizeFilterFuncs, isFilter, err := getVolumeFilterFuncs(filters)
+	if err != nil {
+		return err
+	}
+	if len(sizeFilterFuncs) > 0 && quiet {
+		logrus.Warn("cannot use --filter=size and --quiet together, ignoring --filter=size")
```

Comment:
Why cannot?

Reply:
Merging this PR, but it would be nice if we can relax this restriction

Reply:
https://github.com/containerd/nerdctl/blob/49885ac2aad077f7e1fd10899e3b8b647640fa0c/cmd/nerdctl/volume_ls.go#L75-L78
According to previous behavior, if users enable `--quiet`, the `--size` will be disabled. And the Size field of the Volume struct is left to 0. In that case, `--filter=size` matches no volume. So I restrict those two args not to be together.